examples/model_card_generator/census_income_constants.py

```diff
@@ -0,0 +1,28 @@
+
```

Comment:
You were right about these files not being needed to run the example. I'm starting to think they might have been commited by mistake. We may want to consider adding them to the gitignore.

Reply:
I removed those notebook generated files